pubspec.lock

```diff
@@ -236,4 +380,4 @@ packages:
     version: "0.3.0"
 sdks:
   dart: ">=3.2.5 <4.0.0"
-  flutter: ">=3.3.0"
+  flutter: ">=3.10.0"
```

Comment:
More so for everyone, make sure to keep track of the versions, if there is an updated version in the repo make sure your system matches that version as well!